### PR TITLE
Vignettes

### PR DESCRIPTION
**Pull request by abhiket**
Made Vignettes for the package and also the site using pkgdown

### CODE REVIEW

File: R/extract_lyrics.R [hunk 1]
```diff
@@ -9,37 +9,36 @@
...
-    lyrics <- gsub("\\[[^][]*]"," ",stripped_htmltext)
-    lyrics <- gsub("([a-z])([A-Z])", "\\1 \\2", lyrics)
-    lyrics <- gsub(" Lyrics ", " ", lyrics)
-    lyrics
-}
+     lyrics <- gsub("\\[[^][]*]"," ",stripped_htmltext)
+     lyrics <- gsub("([a-z])([A-Z])", "\\1 \\2", lyrics)
+     lyrics <- gsub(" Lyrics ", " ", lyrics)
+     lyrics
+ }
```
Comment (artanzand) [inline R/extract_lyrics.R]: I am not sure why a single space has been added to these lines. Could you please remove them to original shape?

File: R/extract_lyrics.R [hunk 2]
```diff
@@ -9,37 +9,36 @@
 #' @examples
 #' extract_lyrics( "22", "Taylor Swift")
 extract_lyrics <- function(song_title, artist) {
-
     if ( stringr::str_squish(song_title) == "" | stringr::str_squish(artist) == 0){
         stop("Empty input")
-    }
+   }
```
Comment (artanzand) [inline R/extract_lyrics.R]:
```suggestion
    }
```

File: README.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ lyrics, clean text and generate a word cloud.
 |----------------|----------------------------------------------------------|-----------|---------------------------------------------------------------------------------------------------------------------------------|
 | download_data  | `dataset`, `columns`                                     | Dataframe | Downloads dataset from `kaggle dataset` and extract `columns` from csv file                                                     |
 | extract_lyrics | `song_title`, `artist`                                   | String    | Extracts song lyrics of a song `song_title` by `artist`                                                                         |
-| clean_text     | `text`, `bool_contra_dict`                               | String    | Cleans up the `lyrics` by removing special characters, html tags, \#tags, contraction words and convert everything to lower case |
+| clean_text     | `text`, `bool_contra_dict`                               | String    | Cleans up the `lyrics` by removing special characters, html tags, #tags, contraction words and convert everything to lower case |
```
Comment (artanzand) [inline README.md]:
```suggestion
| clean_text     | `text`, `bool_contra_dict`                               | String    | Cleans up the `lyrics` by removing special characters, html tags, \#tags, contraction words and convert everything to lower case |
```

File: README.md [hunk 4]
```diff
@@ -59,7 +59,7 @@ The rlyrics packages contains the following four functions:
     with columns song title, artist and lyrics.
 
 3.  `clean_text()` The lyrics extracted from `extract_lyrics()` are not
-    clean. It removes special characters, html tags, \#tags, contraction
+    clean. It removes special characters, html tags, #tags, contraction
```
Comment (artanzand) [inline README.md]:
```suggestion
    clean. It removes special characters, html tags, \#tags, contraction
```

File: README.md [hunk 5]
```diff
@@ -75,7 +75,7 @@ dataframe with `columns` argument.
 
 To use the Kaggle API, sign up for a Kaggle account at Kaggle. Then go
 to the ‘Account’ tab of your user profile
-(<https://www.kaggle.com/>\<username\>/account) and select ‘Create API
+(<https://www.kaggle.com/>\<username>/account) and select ‘Create API
```
Comment (artanzand) [inline README.md]:
```suggestion
(<https://www.kaggle.com/>\<username\>/account) and select ‘Create API
```

**Review by artanzand — COMMENTED** [review]
Well done! I had some comments for improvement.

File: R/extract_lyrics.R (hunk 1 above)
Comment (abhiket) [reply R/extract_lyrics.R]: Valid point, the same has been rectified

**Review by artanzand — APPROVED** [review]
Well done! Approving to check the vignettes on the main branch.

**Pull request merged**